book/menus.md

```diff
@@ -4,13 +4,12 @@
 <a href="/d9book">home</a>
 </h3>
 
-
 - [Menus](#menus)
   - [Dynamically change menu items with hook_preprocess_menu](#dynamically-change-menu-items-with-hook_preprocess_menu)
   - [Permanently update menu links in a hook_update using entityQuery](#permanently-update-menu-links-in-a-hook_update-using-entityquery)
   - [Add menu items with hook_update](#add-menu-items-with-hook_update)
   - [Permanently modify or delete menu items with hook_update](#permanently-modify-or-delete-menu-items-with-hook_update)
-  - [Peer up a menu to its parents to see if it is a child of a content type](#peer-up-a-menu-to-its-parents-to-see-if-it-is-a-child-of-a-content-type)
+  - [Pair up a menu to its parents to see if it is a child of a content type](#peer-up-a-menu-to-its-parents-to-see-if-it-is-a-child-of-a-content-type)
```

Comment:
Actually the idea is to peer up the menu chain to find the parent.  Perhaps another word might work better?

Reply:
Of course folks in the US don't generally use the word peer..

Reply:
I read this a few times and thought "I wonder if he means 'pair'?". From what you are saying here you mean more like "look" or "traverse", right?
Of course, I could just be an idiot :D